osquery/tables/system/windows/registry.cpp

```diff
@@ -72,6 +72,21 @@ void explodeRegistryPath(const std::string& path,
   rKey = osquery::join(toks, kRegSep);
 }
 
+Status sanitizeRegistryStrings(const DWORD dataType,
+                               BYTE* dataBuff,
+                               DWORD dataSize) {
+  auto status = Status(0, "OK");
+  if (dataBuff != NULL && dataSize != NULL) {
```

Comment:
It's actually an XOR, if they are both null it's fine. it's only when one si null and the other isn't that I can't do anything. Will fix.